### ISSUE TITLE
Without External Bluetooth Device

### ISSUE THREAD

**Issue by maidis** [BeardoGuy/GamepadBT#2]
Great project. Thank you very much. I have already ordered the hardware you mentioned, but I wonder if this project can be done using the computer's own Bluetooth adapter without the need for extra hardware.

A similar project:

https://github.com/Diallomm/hack_GamesirT1d

**Comment by BeardoGuy** [comment]
Yes, it can be done. Please have a look: https://github.com/BeardoGuy/GamepadBT-Server
